pkg/playground/speed.js

```diff
@@ -212,17 +212,17 @@ function spawn() {
     channel.stream(data => {
         console.log("spawn: stream block length", data.length);
         total += data.length;
-        document.getElementById("spawn-output").innerHTML = data;
+        document.getElementById("spawn-output").textContent = data;
     });
     channel.then(() => {
         console.log("spawn: command finished successfully");
         stop();
-        document.getElementById("spawn-result").innerHTML = "success";
+        document.getElementById("spawn-result").textContent = "success";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19047-20230630-121653-41e6f122-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)